### ISSUE TITLE
Follow PSR-2 code style

### ISSUE THREAD

**Issue by CasperLaiTW** [InfyOmLabs/laravel-generator#11]
Should *.stub file or generator creating file be follow psr-2?

Ex:
[migration.stub](https://github.com/InfyOmLabs/laravel-generator/blob/master/templates/migration.stub#L16)

Convert 1 tab to 4 space ?

**Comment by codingquark** [comment]
Also, for constructor, we cannot omit accessibility declaration according to PSR2. So, current code `function __construct` should be  `public function __construct`

**Comment by mitulgolakiya** [comment]
fixed.
